src/haproxy/address.go

```diff
@@ -0,0 +1,66 @@
+package haproxy
```

Comment:
If PROXY protocol isn't tied to HAProxy specifically, we should call this `proxyprotocol` or something more generic.

I realise that `github.com/pires/go-proxyproto` is imported with that name currently, but a different alias could be used.

Reply:
renamed to `proxyprotocol`